settings/index.html

```diff
@@ -197,6 +197,11 @@ <h1 id="title"></h1>
             <td>Enter a list of domains you don't want this script running on.</td>
             <td><textarea id="blackList" name="blackList" onchange="onChange(this)" cols="30" rows="5"></textarea></td>
           </tr>
+          <tr id="whiteListModeRow">
+            <td><b>whiteListMode</b></td>
+            <td>Turn on whitelist mode.</td>
+            <td><input id="whiteListMode" name="whiteListMode" false="" type="checkbox" onclick="onChange(this)"></td>
```

Comment:
Where does this `false=""` come from?

Reply:
Also, I'd change the description of `blackList`, to explain what it does when whitelist mode is turned on.

Reply:
I directly copied the similar settings below to use, that `false = ""` is the default setting for all `false` settings, isn't it?

Reply:
Oh, then you copied my mistake. `false=""` is an invalid attribute and can be removed from the file.